### PR TITLE
Implemented signup-card component

### PR DESCRIPTION
**Pull request by martinmags**
Displays the login and signup-card components. No functionality. Just the visual of the components.

### CODE REVIEW

File: src/components/login-card/index.jsx [hunk 1]
```diff
@@ -2,21 +2,17 @@ import React from 'react';
...
-      <Card id="card">
-        <p id="card-header">Login</p><br/>
-        <Input id="email-input" placeholder="Email" />        
+      <Card className="card-login">
+        <p style={{fontSize: 22}}>Login</p><br/>
+        <Input style={{marginBottom: 5}} placeholder="Email" />        
         <Input placeholder="Password" /><br/><br/>
-        <Button id="d4sd-btn" type="primary" block>Login</Button> 
+        <Button className="d4sd-btn" block>Login</Button> 
         <br/><br/>
```
Comment (leestanley) [inline src/components/login-card/index.jsx]: Self-enclosing tag

File: src/components/login-card/style.less [hunk 2]
```diff
@@ -2,32 +2,28 @@
 @import '../../styles/fonts.css';
 @import '../../styles/mixins.less';
 
-#card{
+.card-login{
```
Comment (leestanley) [inline src/components/login-card/style.less]: Nest css classnames

File: src/components/signup-card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import './style.less';
+
+const SignupCard = (props) => {
+    return (
+      <Card className="card-signup">
+        <p style={{fontSize: 22}}>Sign Up</p><br/>
+        <Button className="google-btn" icon="google" block>Continue with Google</Button>
+        <Button className="fb-btn" icon="facebook" block>Continue with Facebook</Button>
+        <br/>
+        <p id="or-txt">OR</p>
```
Comment (leestanley) [inline src/components/signup-card/index.jsx]: leftover id

File: src/components/signup-card/style.less [hunk 4]
```diff
@@ -0,0 +1,43 @@
+@import '../../styles/vars.less';
+@import '../../styles/fonts.css';
+@import '../../styles/mixins.less';
+
+// Signup Card 
+.card-signup{
```
Comment (leestanley) [inline src/components/signup-card/style.less]: nest css classnames

File: src/index.js [hunk 5]
```diff
@@ -2,12 +2,15 @@ import React from 'react';
...
 
-import LoginCard from "./components/login-card"
+import LoginCard from "./components/login-card";
+import SignupCard from "./components/signup-card";
 
 const App = () => {
     return (
         <div>
             <LoginCard />
+            <br/>
```
Comment (leestanley) [inline src/index.js]: remove br

File: src/components/signup-card/style.less [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ .google-btn{
+  background-color: #DC4E41;
+  color: white;
+  margin-bottom: 5px;
+  border:none;
+}
+
+.d4sd-btn{
+  background-color: #4497FF;
+  color: white;
```
Comment (leestanley) [inline src/components/signup-card/style.less]: use @white from the vars file

File: src/components/signup-card/style.less [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  border: none;
+}
+
+.fb-btn{
+  background-color: #4267B2;
+  color: white;
```
Comment (leestanley) [inline src/components/signup-card/style.less]: use @white

File: src/components/signup-card/style.less [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  margin-bottom: 10px;
+  border: none;
+}
+
+#or-txt{
+  text-align: center;
+  font-weight: 500;
+  font-size: 15px;
+  color: #828282;
```
Comment (leestanley) [inline src/components/signup-card/style.less]: use color from the vars file

File: src/components/login-card/style.less [hunk 9]
```diff
@@ -2,32 +2,29 @@
...
-
-#email-input{
-  margin-bottom: 5px;
-}
+  /**
+   * Buttons have duplicate styling. 
+   * TODO: Remove duplicate styling into a generic button styling.
+   */
+  .d4sd-btn{
+    background-color: #4497FF;
```
Comment (leestanley) [inline src/components/login-card/style.less]: remove magic colors - use variable from vars file

File: src/components/login-card/style.less [hunk 10]
```diff
@@ -2,32 +2,29 @@
...
-#card-header{
-  font-size: 22px;
-}
-
-#d4sd-btn{
-  background-color: #4497FF;
-  color: white;
-}
+  .fb-btn{
+    background-color: #4267B2;
```
Comment (leestanley) [inline src/components/login-card/style.less]: magic color

File: src/components/login-card/style.less [hunk 11]
```diff
@@ -2,32 +2,29 @@
...
+    color: white;
```
Comment (leestanley) [inline src/components/login-card/style.less]: magic color

File: src/components/login-card/style.less [hunk 12]
```diff
@@ -2,32 +2,29 @@
...
+  }
 
-#fb-btn{
-  background-color: #4267B2;
-  color: white;
+  .google-btn{
+    background-color: #DC4E41;
```
Comment (leestanley) [inline src/components/login-card/style.less]: magic color

File: src/components/login-card/style.less [hunk 13]
```diff
@@ -2,32 +2,29 @@
...
+    margin-bottom: 5px;
+    color: white;
```
Comment (leestanley) [inline src/components/login-card/style.less]: magic color

File: src/components/signup-card/index.jsx [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+            </Col>
+            <Col span={12}>
+              <Input placeholder="Last Name" />
+            </Col>
+          </Row>
+          <Input style={{marginBottom: 5}} placeholder="Email" /><br/><br/>
+          <Input placeholder="Password" /><br/><br/>
+        </Input.Group>
+        <Button className="d4sd-btn" type="primary" block>Sign Up</Button> 
+        <br/><br/>
```
Comment (leestanley) [inline src/components/signup-card/index.jsx]: remove br

File: src/components/signup-card/style.less [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+@import '../../styles/fonts.css';
+@import '../../styles/mixins.less';
+
+// Signup Card 
+.card-signup{
+  width: 40vw;
+  height: 70vh;
+  .flexCenter(column);
+  .cardBorder;
+  font-size: 22px;
```
Comment (leestanley) [inline src/components/signup-card/style.less]: store in vars file

File: src/components/signup-card/style.less [hunk 16]
```diff
@@ -0,0 +1,44 @@
+@import '../../styles/vars.less';
+@import '../../styles/fonts.css';
+@import '../../styles/mixins.less';
+
+// Signup Card 
+.card-signup{
+  width: 40vw;
+  height: 70vh;
```
Comment (leestanley) [inline src/components/signup-card/style.less]: magic numbers - add to vars file

File: src/components/signup-card/index.jsx [hunk 17]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Card, Input, Button, Col, Row } from 'antd';
+import './style.less';
+
+const SignupCard = (props) => {
+    return (
+      <Card className="card-signup">
+        <p style={{fontSize: 22, marginBottom: 10}}>Sign Up</p>
```
Comment (stevemacn) [inline src/components/signup-card/index.jsx]: Why use inline styles here and not less?

File: src/index.js (hunk 5 above)
Comment (martinmags) [reply src/index.js]: done

File: src/components/signup-card/index.jsx (hunk 3 above)
Comment (martinmags) [reply src/components/signup-card/index.jsx]: yuhhh

File: src/components/login-card/style.less (hunk 2 above)
Comment (martinmags) [reply src/components/login-card/style.less]: yuhhhh yeet

File: src/components/signup-card/style.less (hunk 4 above)
Comment (martinmags) [reply src/components/signup-card/style.less]: yuhh yuh skkkrrrt

File: src/components/signup-card/style.less (hunk 6 above)
Comment (martinmags) [reply src/components/signup-card/style.less]: Yuhhh yaat yee yeet

File: src/components/signup-card/style.less (hunk 8 above)
Comment (martinmags) [reply src/components/signup-card/style.less]: yuuhhh yeet skrrrt

File: src/components/login-card/style.less (hunk 11 above)
Comment (martinmags) [reply src/components/login-card/style.less]: magic mike

File: src/components/signup-card/index.jsx (hunk 14 above)
Comment (martinmags) [reply src/components/signup-card/index.jsx]: brrrr skrrrt

File: src/components/signup-card/index.jsx (hunk 17 above)
Comment (martinmags) [reply src/components/signup-card/index.jsx]: removed all inlines c:

File: src/components/signup-card/style.less (hunk 15 above)
Comment (martinmags) [reply src/components/signup-card/style.less]: Dooooone yaaaa yeet

File: src/components/signup-card/style.less (hunk 7 above)
Comment (martinmags) [reply src/components/signup-card/style.less]: skrrrt

**Pull request merged**